### DISCUSSION TITLE
Preemption by Colab

### DISCUSSION THREAD

**Discussion by jw9730** [seominjoon/kaist-ai605#38] [Q&A] [answered]
Hi professor,

I am consistently being preempted by Colab, with the message "GPU 백엔드에 연결할 수 없음. 현재 Colab의 사용량 제한으로 인해 GPU에 연결할 수 없습니다.". It seems that I am apparently blacklisted by Google...

As an alternative, may I run the code at local environment and capture & attach the results in the text section of the notebook? I am afraid that this will leave the execution results blank.

**Comment by seominjoon** [comment, marked as answer]
Yes, you can submit a notebook file as well.
